### PR TITLE
Added security_group_exporter.rb

### PR DESCRIPTION
**Pull request by ajvb**
Used for exporting and then re-importing security groups and there rules.

### CODE REVIEW

File: openstack/migrate-helpers/security_group_exporter.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# Note: As of now, the script just ignores the `default` security group.
+#
+# Original Author: AJ Bahnken / ajvb
+#
+
+require 'json'
+
+# Create output folder
+todays_date = Time.now.strftime "%Y-%m-%d"
+system 'mkdir', "secgroup_exporter_#{todays_date}"
```
Comment (mcasper) [inline openstack/migrate-helpers/security_group_exporter.rb]: If we're going to use Ruby for this, we should either use the FileUtils module (preferred) or the shell syntax.

FileUtils

``` Ruby
require 'fileutils'

FileUtils.mkdir("secgroup_exporter_#{todays_date}")
```

Shelling out

``` Ruby
`mkdir secgroup_exporter_#{todays_date}"`
```

File: openstack/migrate-helpers/security_group_exporter.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+secgroup_importer_script = File.new("secgroup_exporter_#{todays_date}/secgroup_importer.sh", "w")
```
Comment (mcasper) [inline openstack/migrate-helpers/security_group_exporter.rb]: Non-Blocking: If we wanted this to be a little prettier, you could use the File.open block syntax:

``` Ruby
File.open("new_file", "w") do |script|
  script.puts "My stuff"
end
```

Which will also close the file at the end of the block.

File: openstack/migrate-helpers/security_group_exporter.rb (hunk 1 above)
Comment (ajvb) [inline openstack/migrate-helpers/security_group_exporter.rb]: @mcasper I'll convert it to use FileUtils.

Comment (mcasper) [inline openstack/migrate-helpers/security_group_exporter.rb]: <3

**Comment by mcasper** [comment]
Looks great! +1

**Comment by ajvb** [comment]
@mcasper awesome thanks, going to convert it to use the File.open block syntax and add a few more comments then I'll merge it in.

**Pull request merged**